.github/workflows/publish_website.yml

```diff
@@ -44,30 +44,25 @@ jobs:
         with:
           python-version: 3.x
           cache: "pip"
-      - id: "gcp-auth"
-        name: "Authenticating to Google Cloud"
-        uses: "google-github-actions/auth@v1"
-        with:
-          token_format: "access_token"
-          credentials_json: "${{ secrets.IREE_OSS_GITHUB_RUNNER_BASIC_TRUST_SERVICE_ACCOUNT_KEY }}"
-          create_credentials_file: false
       - name: Installing dependencies
         run: |
           pip install -r docs/website/requirements.txt
           pip install requests
+          sudo apt update
+          sudo apt install -y ninja-build
+          echo "CC=clang" >> $GITHUB_ENV
+          echo "CXX=clang++" >> $GITHUB_ENV
```

Comment:
Do these need to be piped to `$GITHUB_ENV` or could these be set as environment variables for a single workflow, like described in https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-environment-variables-for-a-single-workflow?

Reply:
@saienduri had a similar suggestion at https://github.com/iree-org/iree/pull/18422#discussion_r1742752372. I can update all workflows. A few only set the environment conditionally (based on the platform/compiler choice), but most want it set at the whole job level.